src/trainers/trainer-service.js

```diff
@@ -0,0 +1,50 @@
+// declare URL where server listens for HTTP requests
+const TRAINERS_URL = "http://localhost:8080/api/trainers"
```

Comment:
is your port on 8080? The backend should be configured to 8000

Reply:
follow the api endpoints to a t: `"http://localhost:8080/trainers"

Reply:
nope this is from the homework, thanks for the clarification

Reply:
ye, you can use the http endpoints provided in the [Trainer section](https://github.com/CookieComputing/db-pokedex-backend#trainer-api) as well as the move and pokemonInfo sections